src/plugins/plot/stackedPlot/StackedPlot.vue

```diff
@@ -271,23 +271,23 @@ export default {
       this.compositionObjects = [];
     },
 
-    exportJPG() {
+    exportJPG(filename) {
       this.hideExportButtons = true;
       const plotElement = this.$el;
+      filename = filename ?? `${this.domainObject.name} - stacked-plot`;
```

Comment:
Can we encode these file names with `encodeURIComponent` or something? I've been burned by this in the past...